parsl/executors/workqueue/executor.py

```diff
@@ -860,7 +860,11 @@ def _work_queue_submit_wait(*,
                     logger.debug("Sending executor task {} with command: {}".format(task.id, command_str))
                     t = wq.Task(command_str)
                 else:
-                    t = wq.RemoteTask("run_parsl_task", "parsl_coprocess", task.map_file, task.function_file, task.result_file)
+                    t = wq.RemoteTask("run_parsl_task",
+                                      "parsl_coprocess",
+                                      os.path.basename(task.map_file),
+                                      os.path.basename(task.function_file),
+                                      os.path.basename(task.result_file))
                     t.specify_exec_method("direct")
```

Comment:
As an aside&nbsp;&mdash;&nbsp;so not strictly pertinent to this PR&nbsp;&mdash;&nbsp;every time I see `os.path.*`, I cringe a little.  A low-hanging fruit task at some point: transition to `pathlib.Path` where feasible.